src/components/Article.js

```diff
@@ -0,0 +1,20 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+
+export const Article = (props) => {
+  const { title, text, date } = props;
+
+  return (
+    <Fragment>
```

Comment:
use a short form of a fragment
```suggestion
    <>
```